rolemanager/module.py

```diff
@@ -0,0 +1,128 @@
+from typing import List
+
+import discord
+from discord.ext import commands
+
+from pie import utils, check, i18n
+from pie.utils.objects import ConfirmView, ScrollableEmbed
+
+_ = i18n.Translator("modules/fsi").translate
+
+
+class RoleManager(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    # HELPER FUNCTIONS
+    def _create_embeds(ctx, title: str, description: str) -> List[discord.Embed]:
+        elements = []
+        """Create embed for member list.
+        Args:
+            ctx: Command context.
+            option: Item's title.
+            description: list of items.
+        Returns: :class:`discord.Embed` information embed
+        """
+        chunk_size = 15
+
+        for i in range(0, len(description), chunk_size):
+
+            page = utils.discord.create_embed(
+                author=ctx.author,
+                title=title,
+                description="\n".join(description[i : i + chunk_size]),
+            )
+
+            list().append(description[i : i + chunk_size])
+
+            elements.append(page)
+
+        return elements
+
+    def _get_intersection(role_base, role_remove):
```

Comment:
There should be a type hint `)-> Set[discord.Member]:`

Also you have to `from typing import Set`

Reply:
Type hint added.